graph/test/test_graph.c

```diff
@@ -0,0 +1,16 @@
+#include "error.h"
+#include "graph.h"
+#include "test_factories.h"
+#include "pdls.c"
+
+static rage_Error test_graph() {
+    rage_NewGraph new_graph = rage_graph_new(44100);
+    RAGE_EXTRACT_VALUE(rage_Error, new_graph, rage_Graph * graph)
+    rage_Error rv = rage_graph_start_processing(graph);
+    if (!RAGE_FAILED(rv)) {
```

Comment:
So, if we fail to start processing it's ok for this test to pass?

Reply:
If we fail to start processing the test fails doesn't it?
This if block doesn't execute, we clean up and exit with the error, no?

Reply:
Oh yeah, missed the return, my bad.